test/Elastic.Apm.AspNetCore.Tests/TransactionNameTests.cs

```diff
@@ -23,7 +24,10 @@ public class TransactionNameTests : IClassFixture<WebApplicationFactory<Startup>
 		public TransactionNameTests(WebApplicationFactory<Startup> factory)
 		{
 			_factory = factory;
-			_agent = new ApmAgent(new TestAgentComponents(payloadSender: _payloadSender));
+
+			AgentSingletonUtils.EnsureInstanceCreated();
```

Comment:
Is this needed? I think there is no relationship between what this does to the static instance and the `_agent = new ApmAgent` we create in the next line. - but fine by me, in https://github.com/elastic/apm-agent-dotnet/issues/180 we can come back to this.

Reply:
I've added a comment to clarify.
I'm not sure #180 is related to this - here it's all about `CurrentExecutionSegmentsContainer`.

Reply:
Oh, now I see... 👍